variables.tf

```diff
@@ -578,3 +578,32 @@ variable "csi_driver_config" {
   description = "Enable csi-driver configurations (true/false)"
   default     = true
 }
+
+################################################################
+# Image upload variables ( used only for uploading image from 
+# cloud Object storage to ibmcloud)
+################################################################
+variable "import_image" {
+  type        = bool
+  description = "Set to true to upload image to ibmcloud from Cloud Object Storage."
+  default     = false
+}
+
+variable "bucket_region" {
+  type        = string
+  description = "Name of the region where the object storage bucket is available."
+  default     = ""
+}
+
+variable "image_filename" {
+  type        = string
+  description = "Name of the object file. This file is expected to be in .owa.gz format"
+  default     = ""
```

Comment:
Can we have the default to the latest image file? Empty value will be invitation for errors.